app/views/teams/_team_form.html.haml

```diff
@@ -1,6 +1,7 @@
 =form_for @team, :html => { :class => "form-horizontal"} do |f|
   %legend= t('teams.edit.legend')
-  %label.label-custom= t('teams.edit.note')
+  - unless @game.request_team_location && !@game.location_required
```

Comment:
Instead of just showing no help when the game location is not required, should be put the required fields in Bold like we do on the registration page? This seems a little unclear when the game location is not required.